### ISSUE TITLE
Support for Indian and other similar number formats

### ISSUE THREAD

**Issue by apapirovski** [adamwdraper/Numeral-js#19]
Was just talking to a friend about this and in parts of India (or maybe everywhere in India) they format numbers like so: 1,00,00,000 (separator after hundreds until the last thousand) which right now, I believe, is impossible to re-create. There are probably other languages too that format their numbers like this or even more differently.

This is probably a substantial change to how Numeral-js does things. To solve, I assume Numeral-js would need to allow functions instead of objects for the "separator" property of a language which would then allow people to write their own custom formatters. The other possibility is allowing them to specify a custom format like so: '00,00,00,00,00,00,000.00' or something, but that seems kind of unwieldy since you have to make it arbitrarily big to make sure it always works.

**Comment by MSCAU** [comment]
Yes, I need Indian currency formatting and will need to roll my own solution...

**Comment by TimBailey-pnk** [comment]
Any progress on this?
Thanks

**Comment by ooooak** [comment]
+1 this. Everyone in India uses that format

**Comment by sivarajd** [comment]
The number format specified above incorrect, and is not as simple. The actual format used is a repeating pattern of groups of 3;2;2.  I.e., to write hundred trillion (100,000,000,000,000 or 10E14), you would write as 1,00,00,000,00,00,000. It would be read as one crore crore. One crore basically is 10 million and has seven zeros which are grouped together.

**Comment by DilipVxceed** [comment]
any progress on this ? as kibana use numeral-js , and i have requirement where i want show data abbreviation 
as 1 crore instead of  10 million

**Comment by ooooak** [comment]
@DilipVxceed I ended up creating my own package called inrformat. 

npm: https://www.npmjs.com/package/inrformat
source:  https://github.com/ooooak/inrformat

**Comment by Itsmeak0910** [comment]
@ooooak can you tell me how can i use your package in kibana..

**Comment by pratik227** [comment]
Hi,

I just forked the repo and made few changes in it and published my own package. Now it supports India currency.

Checkout - https://www.npmjs.com/package/numeral-v2

Doc - https://numeraljs.netlify.app/

Thanks,
Pratik Patel
